src/member/member.service.ts

```diff
@@ -342,4 +343,11 @@ export class MemberService {
   async deleteMemberByEmail(appId: string, email: string): Promise<DeleteResult> {
     return this.memberInfra.deleteMemberByEmail(appId, email, this.entityManager);
   }
+
+  async getMemberTasks(memberId: string): Promise<Array<MemberTask>> {
+    const tasks = this.entityManager.getRepository(MemberTask).find({
+      where: { member: { id: memberId } },
+    });
```

Comment:
Business logic is written in the service layer, and data queries are written in the infra layer. So it should be written in `member.infra.ts`.

Reply:
updated, thanks for reminder.